### DISCUSSION TITLE
I have a question about a string

### DISCUSSION THREAD

**Discussion by gaelcoral** [ReactiveDrop/reactivedrop_translations#1416] [Q&A] [answered]
I was translating the strings from the reactivedrop_latam.txt file and I came across this string:
`"asw_pda_owner_name_c_s_wiston_beaufort"		"C.S. Wiston Beaufort"`
What does the abbreviation C.S. mean? Since, unlike other strings, this one doesn't say anything. 
The only thing that comes to mind is that it could be Chief of Staff.

**Comment by BenLubar** [comment, marked as answer]
Chief Surgeon

https://github.com/ReactiveDrop/reactivedrop_translations/blob/master/resource/mail/nh_officemail_wistonbeaufort_english.txt

**Reply by gaelcoral** [reply]
Tysm!
